config/brakeman.ignore

```diff
@@ -1,5 +1,25 @@
 {
   "ignored_warnings": [
+    {
```

Comment:
Instead of ignoring these warnings can you pass in the parameters into the execute commands? Something like this:

```ruby
    result = ActiveRecord::Base.connection.execute("
      SELECT DISTINCT SUBSTRING(sha2_base16, 1, #{RESPONSES_PREFIX_LENGTH * 2}) as prefix
      FROM site_banner_lookups
      WHERE wallet_status != 0
      AND updated_at >= :previous_run'
      ORDER BY prefix desc",
      previous_run: previous_run
    )
```

If that doesn't work [this article has more information](https://blog.saeloun.com/2019/10/28/bind-parameters-in-activerecord-sql-queries.html) about binding parameters.